test/system/065-cp.bats

```diff
@@ -949,9 +949,107 @@ ${randomcontent[1]}" "$description"
     run_podman rm -t 0 -f cpcontainer
 }
 
+@test "podman cp --overwrite file - ctr/ctr" {
+    rand_content_file=$(random_string 50)
+    rand_content_dir=$(random_string 50)
+
+    run_podman run -d --name ctr-file $IMAGE sh -c "echo '$rand_content_file' > /tmp/foo; sleep infinity"
+    run_podman run -d --name ctr-dir  $IMAGE sh -c "mkdir /tmp/foo; echo '$rand_content_dir' > /tmp/foo/file.txt; sleep infinity"
+
+    # overwrite a directory with a file
+    run_podman 125 cp ctr-file:/tmp/foo ctr-dir:/tmp
+    if ! is_remote; then # remote just returns a 500
+        is "$output" ".* error creating \"/tmp/foo\": .*: file exists.*"
+    fi
```

Comment:
Why is the error message different? We should return a reasonable error for the remote client.